cli/internal/core/runer.go

```diff
@@ -146,10 +152,9 @@ func (ivkr _runer) Run(
 	for {
 		select {
 
-		case <-signalChannel:
-			if intSignalsReceived == 0 {
-
-				intSignalsReceived++
+		case <-sigIntChannel:
+			sigIntsReceived++
```

Comment:
Thx for contribution! @apexskier why is sigIntsReceived needed?

Reply:
It's a rename of `intSignalsReceived`, which exists to support a second ctrl+c forcefully stopping execution.

I renamed because it's not used for the new signal handler. I changed the logic slightly here to ensure it's actually counting. Previously it was essentially a boolean flag (which I actually might change it to) since it was only incrementing once.